src/ast/value.rs

```diff
@@ -270,6 +270,27 @@ impl fmt::Display for DateTimeField {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Ord, PartialOrd, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub enum NormalizationForm {
+    NFC,
```

Comment:
Bonus points if you added some comments here (maybe referencing the postgres docs and defining what the `NFC` and `NFD`, etc acronyms meant

I think you could copy from your very nice PR description 🙏

Reply:
Good idea; done :)